8_ephem_bot.py

```diff
@@ -13,42 +13,55 @@
 
 """
 import logging
-
+import ephem
+from datetime import date
 from telegram.ext import Updater, CommandHandler, MessageHandler, Filters
 
-logging.basicConfig(format='%(name)s - %(levelname)s - %(message)s',
-                    level=logging.INFO,
-                    filename='bot.log')
-
-
-PROXY = {
-    'proxy_url': 'socks5://t1.learn.python.ru:1080',
-    'urllib3_proxy_kwargs': {
-        'username': 'learn',
-        'password': 'python'
-    }
+logging.basicConfig(
+    format="%(name)s - %(levelname)s - %(message)s",
+    level=logging.INFO,
+    filename="bot.log",
+)
+
+planet_dict = {
+    "Mars": ephem.Mars(date.today()),
+    "Venus": ephem.Venus(date.today()),
+    "Saturn": ephem.Saturn(date.today()),
+    "Jupiter": ephem.Jupiter(date.today()),
+    "Neptune": ephem.Neptune(date.today()),
+    "Uranus": ephem.Uranus(date.today()),
+    "Mercury": ephem.Mercury(date.today()),
 }
 
 
 def greet_user(update, context):
-    text = 'Вызван /start'
+    text = "Вызван /start"
     print(text)
     update.message.reply_text(text)
 
 
 def talk_to_me(update, context):
     user_text = update.message.text
     print(user_text)
-    update.message.reply_text(text)
+    update.message.reply_text(user_text)
+
+
+def planets(update, context):
+    planetname = update.message.text.split()[1]
+    isplanet = planet_dict.get(planetname, None)
+    if isplanet != None:
```

Comment:
Проверка на None всегда делается как 
```suggestion
    if isplanet is not None:
```
Но только с None